### ISSUE TITLE
Implement live templates

### ISSUE THREAD

**Issue by kinglozzer** [kinglozzer/idea-silverstripe-plugin#5]
The live templates from the original plugin look like they _should_ work: https://github.com/raket/idea-silverstripe/tree/master/resources/liveTemplates.

In my initial testing they didn’t backtrack far enough during replacement:

```html
<!-- Selecting the completion after this -->
<% if 

<!-- Would result in something like this -->
<%<% if
```

It could be because the templates have to use HTML entities for `<` and `%` characters, and the IDE doesn’t take that into account - in which case it would be a platform bug. Needs more investigation anyway.

**Comment by kinglozzer** [comment]
I think I actually just misunderstood how these are supposed to be implemented - it’s probably a case of making sure they only fire in the right context (i.e. stop them firing within a Silverstripe block `<% `)

**Comment by kinglozzer** [comment]
Released in 0.1.0
